docs/animations/Light.md

```diff
@@ -1,64 +1,68 @@
 # Light
 
-The **Light animation behavior** performs a point light (A point source of light that emits light in all directions) in the middle of a given UIElement. 
-The light behavior is great drawing the user's eye towards a particular pieces of user interface. You set the distance property of the 
-light to determine how bright the light will be. The closer the light source, the more focused it will be, but, will make the overall UI element darker.
-The further away from the light source the more the light will spread over the UIElement.
-
-**NOTE**:  Heavy usage of effects may have a negative impact on the performance of your application. 
+The Light animation behavior performs a point light (A point source of light that emits light in all directions) in the middle of a given UIElement. You set the distance property of the light to determine how bright the light will be. If the the light source is closer then will make the overall UI element darker. ***NOTE**:  Heavy usage of effects may have a negative impact on the performance of your application.* 
```

Comment:
remove double "the". This line sounds weird too. Maybe rephrase to: The closer the light source, the darker the UI element will be